### PR TITLE
Add a new statistics api

### PR DESCRIPTION
**Pull request by axlbonnet**
issue #261

### CODE REVIEW

File: vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/StatsApiBusiness.java [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        }
+        return new UsersNumber(startDate.atStartOfDay(), endDateTime, usersRegisteredNumber);
+    }
+
+
+    public UsersList getAllUsers() throws ApiException {
+        return getUsersList(new UserSearchCriteria());
+    }
+
+    public UsersList getAllUsersBetweenDate(
```
Comment (camarasu) [inline vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/StatsApiBusiness.java:87]: Any particular reason from not calling this "getAllUsersFromDate" ?

Comment (axlbonnet) [reply vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/StatsApiBusiness.java:87]: Copy/paste errror....

File: vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/StatsApiBusiness.java [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+            @NotNull String startDateString) throws ApiException {
+        return getUsersList(new UserSearchCriteria()
+                .withRegistrationStart(parseDate(startDateString)));
+    }
+
+    public UsersList getAllUsersBetweenDate(
```
Comment (camarasu) [inline vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/StatsApiBusiness.java:89]: Just a minor remark, you use both BetweenDate and BetweenDates. Would it be possible to use the second one everywhere ?

Comment (axlbonnet) [reply vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/StatsApiBusiness.java:89]: Yes, it would be better indeed !

**Comment by camarasu** [comment]
Great, thanks !

**Pull request merged**